### PR TITLE
Feature/http port

### PR DESCRIPTION
**Pull request by chrisabrams**
The default server port is 3000 but when developing sometimes changing the port in the config is not an available option. Defining the environment variable `HTTP_PORT` is another option.

### CODE REVIEW

File: js/config.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Object.defineProperty(exports, "__esModule", { value: true });
 const _0x_js_1 = require("0x.js");
 // Network port to listen on
-exports.HTTP_PORT = 3000;
+exports.HTTP_PORT = process.env.HTTP_PORT || 3000;
```
Comment (fabioberger) [inline js/config.js:5]: @chrisabrams can we make `HTTP_PORT` and config exported from `config.ts`? We've opted for keeping all configurations in that one place, rather then splitting between there and environment variables.

File: README.md [hunk 2]
```diff
@@ -91,6 +91,9 @@ OR
 yarn start:js
 ```
 
+#### Server Port
+The default server port is 3000 but when developing sometimes changing the port in the config is not an available option. Defining the environment variable `HTTP_PORT` is another option.
+
```
Comment (fabioberger) [inline README.md:96]: Once this is moved to `config.ts`, we can add this setting to the `config` step above.

File: yarn.lock [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 "0x.js@^1.0.4":
   version "1.0.8"
-  resolved "https://registry.yarnpkg.com/0x.js/-/0x.js-1.0.8.tgz#779d0d394ba3e4519a980b1035864f87297b0d52"
+  resolved "http://localhost:4873/0x.js/-/0x.js-1.0.8.tgz#779d0d394ba3e4519a980b1035864f87297b0d52"
```
Comment (fabioberger) [inline yarn.lock:7]: Looks like you're using a local registry, however you cannot push these changes since other dev's don't have your local setup.

**Review by fabioberger — CHANGES_REQUESTED** [review]

File: js/config.js (hunk 1 above)
Comment (LogvinovLeon) [reply js/config.js:5]: I agree that those simple configs that can be configured over ENV variables might be done so.

File: README.md (hunk 2 above)
Comment (LogvinovLeon) [reply README.md:96]: You shouldn't call this section `Server Port` There would be other env configs in the future. Let's call it Configuration and describe that you can configure everything via config.ts or override it with env vars

**Review by LogvinovLeon — CHANGES_REQUESTED** [review]

**Comment by dekz** [comment]
Needed this and a bit more for my own needs, opened #21.

**Pull request closed without merging**